src/server/services/get-template.js

```diff
@@ -11,6 +11,10 @@ const xlsx = require('xlsx')
 const _ = require('lodash')
 const { sheetToJson } = require('../lib/spreadsheet')
 
+// Treasury upload spreadsheet template
+// TODO: Support all treasury export templates
+const PROJECT_BASELINE_TEMPLATE = 'treasury/projectBaselineBulkUpload.xlsx'
```

Comment:
Based on our chat earlier, I think we want to just configure the export templates in code, rather than via env variable.  What do you think?

Reply:
sgtm; i wonder if we want to declare a mapping from type of output to it's filename, vs having a variable name for each of the 16 output templates? then `loadTreasuryTemplate` can take a `type arg for the output template to load

Reply:
That sounds pretty reasonable to me.  This is definitely just a placeholder.  👍